http/src/api_error.rs

```diff
@@ -0,0 +1,452 @@
+    /// Maximum number of guild channels reached (500)
+    MaximumGuildChannelsReached,
+    /// Maximum number of attachments in a message reached (10)
+    MaximumAttachmentsReached,
+    /// Maximum number of invites reached (1000)
+    MaximumInvitesReached,
+    /// Unauthorized. Provide a valid token and try again
+    Unauthorized,
+    /// You need to verify your account in order to perform this action
+    AccountNeedsVerification,
+    /// Request entity too large. Try sending something smaller in size
+    RequestEntityTooLarge,
+    /// This feature has been temporarily disabled server-side
+    FeatureTemporarilyDisabled,
+    /// The user is banned from this guild
+    UserBannedFromGuild,
+    /// Missing access
+    Missingaccess,
+    /// Invalid account type
+    InvalidAccountType,
+    /// Cannot execute action on a DM channel
+    InvalidDMChannelAction,
+    /// Guild widget disabled
+    GuildWidgetDisabled,
+    /// Cannot edit a message authored by another user
+    MessageNotAuthoredByUser,
+    /// Cannot send an empty message
+    EmptyMessage,
+    /// Cannot send messages to this user
+    CannotSendMessageToUser,
+    /// Cannot send messages in a voice channel
+    CannotSendMessagesInVoiceChannel,
+    /// Channel verification level is too high for you to gain access
+    VerificationLevelTooHigh,
+    /// OAuth2 application does not have a bot
+    OAuthApplicationHasNoBot,
+    /// OAuth2 application limit reached
+    OAuthApplicationLimitReached,
+    /// Invalid OAuth2 state
+    InvalidOAuthSstate,
+    /// You lack permissions to perform that action
+    PermissionsLacking,
+    /// Invalid authentication token provided
+    InvalidAuthenticationTokenProvided,
+    /// Note was too long
+    NoteTooLong,
+    /// Provided too few or too many messages to delete. Must provide at least 2 and fewer than 100 messages to delete
+    InvalidMessageDeleteRange,
+    /// A message can only be pinned to the channel it was sent in
+    MessagePinnedInWrongChannel,
+    /// Invite code was either invalid or taken
+    InviteCodeInvalidOrTaken,
+    /// Cannot execute action on a system message
+    InvalidActionOnSystemMessage,
+    /// Invalid OAuth2 access token provided
+    InvalidOAuthAccessToken,
+    /// A message provided was too old to bulk delete
+    MessageTooOldToBulkDelete,
+    /// Invalid form body (returned for both application/json and multipart/form-data bodies), or invalid Content-Type provided
+    InvalidFormBodyOrContentType,
+    /// An invite was accepted to a guild the application's bot is not in
+    InviteAcceptedToGuildBotNotIn,
+    /// Invalid API version provided
+    InvalidApiVersion,
+    /// Reaction was blocked
+    ReactionBlocked,
+    /// API resource is currently overloaded. Try again a little later
+    ApiResourceOverloaded,
+    /// A status code that Twilight doesn't have registered.
+    ///
+    /// Please report the number if you see this variant!
+    Other(u64),
```

Comment:
Would it be possible to write out a warn or info if this is hit?

Reply:
Done